html/components/SimpleHtml.js

```diff
@@ -56,10 +57,14 @@ class SimpleHtml extends PureComponent {
    * video iframe tags in when video format is unsupported
    */
   alterNode(node) {
-    const { style } = this.props;
+    const { customAlterNode, style } = this.props;
 
     const styleAttrib = _.get(node, 'attribs.style', '').trim();
 
+    if (customAlterNode && _.isFunction(customAlterNode)) {
+      return customAlterNode(node, cssObjectToString, cssStringToObject);
+    }
+
```

Comment:
If we pass `customAlterNode` for `img` tags, won't this prevent our current existing alter nodes for table and others to be used?

Reply:
It will, but...
The idea is that you point to specific node IDs & alter only those.
If you point to specific ID only, inside your `customAlterNode` fn, other `img` tags will be altered by `SimpleHtml`.

Reply:
Ah, I see what I've done wrong...
It'll return `null` too early.

Reply:
I understand that, but `alterNode` is going to check if `customAlterNode` exists, if it does, it will execute it and that's it, return stops function execution, it doesn't matter if `customAlterNode` returns `null`, no? Because there's no way `alterNode` can know the ID `customAlterNode` is looking for, it'll just always execute it if it exists for every tag, preventing the other alter functions inside `alterNode` to be executed.

Maybe I'm not getting something but this looks like it just won't ever do anything other than `customAlterNode` when it's defined.

Reply:
I was **nedorečen** again, gotta stop doing it.

I assume my last commit is the solution to your question 'coz you approved it.
By "too early null return" - I meant exactly what you described in comment before.
I've added additional IF statement, if `customAlterNode` returns altered node, it'll return.
If not, if the result of `customAlterNode` is null, it'll continue with the `alterNode` function execution.